srcpkgs/openjdk8/template

```diff
@@ -186,11 +186,17 @@ do_configure() {
 	CXXFLAGS=${CXXFLAGS/-D_FORTIFY_SOURCE=2/}
 
 	# force ELFv2 for ppc64 just in case
+	# with normal -O2 on mipshf there is a too-large jump which causes the
+	# assembler to fail. optomising for size means the jump is made smaller
```

Comment:
"optimising"

And I think it would be best if the comments were right above their respective archs, since there are different special cases here.